search-server/spacewalk-search/spacewalk-search.spec

```diff
@@ -161,115 +120,44 @@ cp -d lib/* $RPM_BUILD_ROOT/%{_prefix}/share/rhn/search/lib
 
 install -p -m 644 src/config/etc/logrotate.d/rhn-search $RPM_BUILD_ROOT%{_sysconfdir}/logrotate.d/rhn-search
 install -p -m 755 src/config/rhn-search $RPM_BUILD_ROOT%{_sbindir}
-%if 0%{?fedora} || 0%{?rhel} >=7 || 0%{?suse_version} >= 1210
 install -p -m 644 src/config/rhn-search.service $RPM_BUILD_ROOT%{_unitdir}
-%else
-install -p -m 755 src/config/rhn-search.init $RPM_BUILD_ROOT%{_initrddir}/rhn-search
-%endif
 install -p -m 644 src/config/search/rhn_search.conf $RPM_BUILD_ROOT%{_prefix}/share/rhn/config-defaults/rhn_search.conf
 install -p -m 644 src/config/search/rhn_search_daemon.conf $RPM_BUILD_ROOT%{_prefix}/share/rhn/config-defaults/rhn_search_daemon.conf
 ln -s -f %{_prefix}/share/rhn/search/lib/spacewalk-search-%{version}.jar $RPM_BUILD_ROOT%{_prefix}/share/rhn/search/lib/spacewalk-search.jar
-%if 0%{?fedora} && 0%{?fedora} >= 21
-sed -i 's/log4j.jar/log4j-1.jar/' $RPM_BUILD_ROOT%{_prefix}/share/rhn/config-defaults/rhn_search_daemon.conf
-%endif
-%if 0%{?suse_version} >= 1315
-sed -i 's/^wrapper.java.classpath.19=.*/wrapper.java.classpath.19=\/usr\/share\/nutch\/nutch-core-1.0.jar/' $RPM_BUILD_ROOT%{_prefix}/share/rhn/config-defaults/rhn_search_daemon.conf
-%endif
 
 # add rc link
 mkdir -p  $RPM_BUILD_ROOT/%{_sbindir}/
-%if 0%{?suse_version} >= 1210
 ln -sf service $RPM_BUILD_ROOT/%{_sbindir}/rcrhn-search
-%else
-ln -sf ../../etc/init.d/rhn-search $RPM_BUILD_ROOT/%{_sbindir}/rcrhn-search
-%endif
-
-# mybatis is build with newer API. This statements needs to stay, until we have a build with 1.5
-export NO_BRP_CHECK_BYTECODE_VERSION=true
 
 %post
-was_running=0
-%if 0%{?suse_version} >= 1210
 %service_add_post rhn-search.service
-%else
-if [ -f /etc/init.d/rhn-search ]; then
-   # This adds the proper /etc/rc*.d links for the script
-   /sbin/chkconfig --add rhn-search
-
-   if /sbin/service rhn-search status > /dev/null 2>&1 ; then
-       was_running=1
-   fi
-fi
-%endif
-
-# Migrate original /usr/share/rhn/search/indexes/*
```

Comment:
In which cases was this index migration being done?

Reply:
A long time ago (somewhen in 2012) the index was moved to a different directory. It is so long ago, that I would say we do not need it anymore.